docs/program/apis/cloud.md

```diff
@@ -33,9 +33,10 @@ Cloud app API methods are only available in the Python SDK.
 
 To use the Viam cloud app API, you first need to instantiate a [`ViamClient`](https://python.viam.dev/autoapi/viam/app/viam_client/index.html#viam.app.viam_client.ViamClient) and then instantiate an [`AppClient`](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient).
 See the following example for reference.
-To find the location secret, go to [Viam app](https://app.viam.com/), and go to the [**Code sample**](https://docs.viam.com/manage/fleet/robots/#code-sample) tab of any of the robots in the location.
-Toggle **Include secret** on and copy the `payload`.
-For the URL, use the address of any of the robots in the location (also found on the **Code sample** tab).
+To find the api key and secret, go to [Viam app](https://app.viam.com/), and navigate to the bottom of your location's page.
+For the URL, use the address of any of the robots in the location (found on the **Code sample** tab).
+
+TODO
```

Comment:
@jr22 or @ohEmily can you confirm for me that we still need to use the location secret for the ViamClient?

Reply:
Nevermind - figured it out myself